src/renderer/page/settings/view.jsx

```diff
@@ -80,15 +84,18 @@ class SettingsPage extends React.PureComponent<Props, State> {
   }
 
   onDownloadDirChange(newDirectory: string) {
-    this.setDaemonSetting('download_directory', newDirectory);
+    this.setDaemonSetting('download_dir', newDirectory);
   }
 
   onKeyFeeChange(newValue: Price) {
+    this.props.setClientSetting(SETTINGS.LOCAL_MAX_KEY_FEE, newValue);
     this.setDaemonSetting('max_key_fee', newValue);
   }
 
   onKeyFeeDisableChange(isDisabled: boolean) {
-    this.setDaemonSetting('disable_max_key_fee', isDisabled);
+    this.props.setClientSetting(SETTINGS.DISABLE_MAX_KEY_FEE, isDisabled);
+    // null is default value passed to clear key fee
+    if (isDisabled) this.setDaemonSetting('max_key_fee');
```

Comment:
We shouldn't keep this as a local setting. It should stay in the daemon only. Then we don't have the same setting being tracked in two places.

Reply:
But it will either be null or a value - then we need to check for the case of a value. I can change this...

Reply:
Yeah.

```
this.setDaemonSetting('max_key_fee', isDisabled ? null : true);
```

I think that will work.

Reply:
Just removed the top part...otherwise, true is not a valid value. When you choose the other option, it sets a LBC/USD amount.  Changes have been pushed.